Dockerfile_Production

```diff
@@ -0,0 +1,23 @@
+FROM node:alpine as build
+
+WORKDIR /app
+
+COPY package.json .
+
+RUN npm install
+
+COPY . .
```

Comment:
Don't we need the docker ignore file since when we copy the all folders we are replacing the node_modules right?